lib/LedgerSMB/Budget.pm

```diff
@@ -246,7 +246,7 @@ sub from_input {
              $line->{amount} =   $input->{"credit_$rownum"}
                                - $input->{"debit_$rownum"};
              $line->{credit} = $line->{amount} if $line->{amount} > 0;
-             $line->{debit}  = $line->{amount} * -1 if $line->{amount} < 0;
+             $line->{debit}  = -$line->{amount} if $line->{amount} < 0;
```

Comment:
@ziuq absolutely agreed. It's definitely one of the things that the Critic tests bring to a code base: a good reason to go over the code once more with a different pair of eyes and intentions. It helps to coax out latent problems.